### DISCUSSION TITLE
Dropdown Tippy Full Width of Target

### DISCUSSION THREAD

**Discussion by lsbyerley** [atomiks/tippyjs#1009] [Q&A] [answered]
Hi.. I am using Tippyjs as a dropdown component and the target element used to trigger the dropdown is fairly long. The elements inside the dropdown are not quite as long so the dropdown does not stretch the full width of the target element (See screenshot below). Adding a `width:100%` to the outer container in the `content` prop does not change anything. How can one achieve this?

![Screen Shot 2021-11-18 at 8 51 41 AM](https://user-images.githubusercontent.com/3066258/142428557-3400c15b-741a-4602-bc6b-88671fc910e8.png)

**Comment by lsbyerley** [comment, marked as answer]
It appears the sameWidth modifier listed here as popper modifier is the answer:
https://popper.js.org/docs/v2/modifiers/community-modifiers/

**Reply by lsbyerley** [reply]
However, there is an issue with the default render props set by tippy (it's hardcoded to 350px and is not allowed to be overridden). So, the width of the dropdown is capped at 350px.

https://github.com/atomiks/tippyjs/blob/ba41007792b2d1e380e329990d49bba229a7f102/src/props.ts#L23

**Reply by atomiks** [reply]
`maxWidth: 'none'` removes that.

**Reply by lsbyerley** [reply]
Not sure how I missed that prop! Thank's for pointing it out!
